### ISSUE TITLE
This tool is awesome thank you

### ISSUE THREAD

**Issue by NZzer0** [SimonvBez/SatisfactoryTrainRearranger#1]
Just wanted to thank you for making this. Works great for me. 
I use it with the Station Master mod and it works fine by the way. 
Guess I need to think of an issue.. OK.. so why dont you list it on satisfactory mods or Nexusmods?

**Comment by SimonvBez** [comment]
Glad to hear it's been helpful to someone! 😄 
I don't even know if this could be posted on those websites since this isn't running inside the game like the usual mods.
I've been meaning to at least post on the Satisfactory subreddit about this, but haven't taken the time for it yet. I really should though!

**Comment by NZzer0** [comment]
yes im sure it can - the goatfungus save editor for no mans sky is on
nexusmods and thats outside the game
Simon
